CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0-rc.1] - 2024-06-24
+
+Part of Patient App refresh project
```

Comment:
Lets follow the changelog spec https://keepachangelog.com/en/1.1.0/

Reply:
should I include the names of all the added icons?